packages/fluentui/react/src/themes/teams-high-contrast/components/Menu/menuVariables.ts

```diff
@@ -18,10 +18,40 @@ export default (siteVars: any): Partial<MenuVariables> => ({
   ),
   color: siteVars.colors.white,
   colorActive: siteVars.colors.black,
+  colorFocus: siteVars.colors.black,
   backgroundColorFocus: siteVars.accessibleCyan,
   backgroundColorActive: siteVars.accessibleCyan,
   primaryBorderColor: siteVars.colors.white,
+  activeUnderlinedColor: siteVars.colors.white,
+  activeUnderlinedBorderBottomColor: siteVars.colors.white,
+  activeUnderlinedWrapperColor: siteVars.colors.white,
+  pointingIndicatorBackgroundColor: 'transparent',
 
   verticalBackgroundColor: siteVars.colors.black,
-  iconOnlyColorActive: siteVars.colors.black
+  verticalBackgroundColorFocus: siteVars.accessibleCyan,
+  iconOnlyColorActive: siteVars.colors.black,
+  iconOnlyColorFocus: 'transparent',
+  iconOnlyColorHover: siteVars.colors.black,
+  iconOnlyWrapperBackgroundColorFocus: siteVars.accessibleCyan,
+  activeIconOnlyWrapperBackgroundColor: siteVars.accessibleCyan,
+  activeUnderlinedPrimaryColor: siteVars.colors.white,
+
+  wrapperColorActive: siteVars.colors.black,
+  wrapperBackgroundColorFocus: siteVars.accessibleCyan,
+  wrapperColorFocus: siteVars.colors.black,
+  primaryWrapperColorHover: siteVars.colors.black,
+  wrapperColorHover: siteVars.colors.black,
+
+  primaryWrapperColorFocus: siteVars.colors.black,
+  primaryWrapperBackgroundColorFocus: siteVars.accessibleCyan,
+  underlinedColorHover: siteVars.colors.white,
+  underlinedWrapperColorHover: siteVars.colors.white,
+
+  iconOnlyBackgroundColorHover: siteVars.accessibleCyan,
```

Comment:
I noticed that all hover styles are changed from accessibleYellow to accessibleCyan in a prev PR. I really hope the changes were intentional..

Reply:
Yep, we discussed this with @pompomon when we compared with `master` and it was really confusing for us. For reference, this was done in #12112.